lib/spack/spack/test/packaging.py

```diff
@@ -191,6 +191,20 @@ def test_packaging(mock_archive, tmpdir):
     buildinfo = bindist.read_buildinfo_file(spec.prefix)
     assert(buildinfo['relocate_textfiles'] == ['dummy.txt'])
 
+    old_dir = '/home/spack/opt/spack'
```

Comment:
Thanks for adding this! Would it be possible to move this into a separate unit test? I'm assuming it could just be moved and no additional setup would be needed.